stories/assets/img/bat.svg

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please optimize this image https://github.com/brave/brave-ui/wiki/How-to-handle-icons